src/test/unit/project-converter_test.ts

```diff
@@ -975,6 +975,35 @@ export function arrowFn() {
       });
     });
 
+    let testName = 'references to namespace objects are rewritten to ' +
+        '`void 0` when they do not start member expressions';
+    test(testName, async () => {
+      setSources({
+        'test.html': `
+          <script>
+            /** @namespace */
+            window.NS = {
+              f: function() {
+                console.log('f', this);
+                this.g();
+              },
+              g: function() { console.log('g'); },
+            };
+          </script>`,
+      });
+      assertSources(await convert({namespaces: ['NS']}), {
```

Comment:
The explicit namespaces argument shouldn't be necessary... I think.... since you've got the `@namespace` in the html.

Reply:
Yup, `namespaces` isn't needed here - removed. Although, removing the `@namespaces` annotation but leaving the `namespaces` argument doesn't work.